src/types/UiaTracing.cpp

```diff
@@ -126,6 +126,140 @@ inline std::wstring UiaTracing::_getValue(const TextUnit unit) noexcept
     }
 }
 
+std::wstring UiaTracing::convertAttributeId(const TEXTATTRIBUTEID attrId) noexcept
+{
+    // Source: https://docs.microsoft.com/en-us/windows/win32/winauto/uiauto-textattribute-ids
+    switch (attrId)
+    {
+    case UIA_AfterParagraphSpacingAttributeId:
+        return L"AfterParagraphSpacing";

```

Comment:
this many strings will contribute badly to the size of the conhost binary for a rarely-used scenario; can we just use numbers and expect the developer to look the attribute up in a table?

Reply:
I like this idea best haha.